quickstarts/lacework/config.yml

```diff
@@ -1,16 +1,29 @@
 id: 8a7a7220-e8ec-4959-b35d-0fe082be8039
 name: lacework
-description: End-to-end security management and diagnostics for major cloud platforms.
-summary: End-to-end security management and diagnostics for major cloud platforms.
+title: Lacework Integration
+summary:  >-
+  Integrate Lacework's security events into the New Relic platform
+
+description: >-
+  Lacework is a comprehensive cloud security platform. We aim to turn security into a data problem and replace the frictionful processes for things like breach investigations or achieving compliance standards with simple, contextualized workflows.
+
+  Technically, our approach is that we replace the traditional 'rules' based mentality that requires you to predict attacker patterns ahead of time with a fully ML based approach.  We baseline what normal user, application and network behaviors look like across your workloads and cloud accounts automatically and then only alert you to deviations from the norm.  This significantly reduces the amount of toil in setting up and maintaining our solution, but also drastically improves the efficacy and amount of security alerts you will receive.
+
+  The following Quickstart brings the curated security alerts for misconfigurations and anomalous security behaviors into the New Relic platform so you can easily triage against your wealth of observability data or easily transition from monitoring to security investigations from a single interface!
+
```

Comment:
```suggestion

For more information or support go to support.lacework.com
```

Reply:
@lars2893 for partner quickstarts we want to add a snippet like the one I suggested.

Reply:
ah was so close! will get that added! Is there any concern with adding an email as well, @polfliet ? Just for the sake of ensuring we give the best turnaround getting it routed to me for now would be ideal.

Reply:
sure, that works!

Reply:
Updated!